validate-crt-with-docker.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+CERT_FILE="`pwd`/generated-certs/crt.pem"
+CERT_FILE_MOUNT_POINT="/run/spid-complaint-certificates-validator/crt.pem"
+
```

Comment:
Before going ahead, I would check if `CERT_FILE` and `CERT_FILE_MOUNT_POINT` exists

Reply:
The mountpoint is fixed and shouldn't be changed by the user, is only there for readability;
the CERT_FILE is checked with `cat` in conjunction with the `set -e`: if the crt.pem doesn't exists the program does as follows:
```
$ ./validate-crt-with-docker.sh
### Starting validation of certificate file at './spid-compliant-certificates/generated-certs/nope.pem':

cat: ./spid-compliant-certificates/generated-certs/nope.pem': No such file or directory
$
```

Should I change the strategy or its fine like this?

Reply:
It's fine